src/index.css

```
@@ -70,14 +84,12 @@ body,
   width: 100%;
 }
 
-html body[data-scroll-locked] {
+body {
   overflow-y: inherit !important;
   overflow-x: inherit !important;
+  overflow: inherit !important;
```

Biome (1.9.4)

[error] Unexpected shorthand property overflow after overflow-y



(lint/suspicious/noShorthandPropertyOverrides)